docs/topics/Voice_Connections.md

```diff
@@ -2,88 +2,167 @@
+		"user_id": "104694319306248192",
+		"session_id": "my_session_id",
+		"token": "my_token"
+	}
 }
 ```
 
-The voice server should respond with an OP2 Ready payload, which informs us of our `ssrc`, the UDP port, encryption modes supported and heartbeat_interval the voice server expects. Using this information, we can finally move on to [Establishing a Voice UDP Connection](#DOCS_VOICE_CONNECTIONS/establishing-a-voice-udp-connection).
+The voice server should respond with an [OP 2 Ready](#DOCS_VOICE_CONNECTIONS/voice-events-voice-op-codes) payload, which informs us of the `ssrc`, UDP port, supported encryption modes, and heartbeat_interval the voice server expects. Using this information, we can finally move on to [Establishing a Voice UDP Connection](#DOCS_VOICE_CONNECTIONS/establishing-a-voice-udp-connection).
 
 ###### Example Voice Ready Payload
 
 ```json
 {
-	"ssrc": 1,
-	"port": 1234,
-	"modes": ["plain", "xsalsa20_poly1305"],
-	"heartbeat_interval": 1
+	"op": 2,
+	"d": {
+		"ssrc": 1,
+		"port": 1234,
+		"modes": ["plain", "xsalsa20_poly1305"],
+		"heartbeat_interval": 1
+	}
 }
 ```
 
 ### Establishing a Voice UDP Connection
 
-Once we receive the properties of a UDP voice server from our OP2 ready payload, we can proceed to the final step of voice connections, which entails establishing and handshaking a UDP connection for voice data. First, we open a UDP connection to the same endpoint we originally received in the [Voice Server Update](#DOCS_GATEWAY/voice-server-update) payload, combined with the port we received in the Voice Ready payload. If required, we can now perform an [IP Discovery](#DOCS_VOICE_CONNECTIONS/ip-discovery) using this connection. Once we've fully discovered our external IP and UDP port, we can then tell the voice websocket what it is, and start receiving/sending data. We do this using the select protocol payload.
+Once we receive the properties of a UDP voice server from our [OP 2 Ready](#DOCS_VOICE_CONNECTIONS/voice-events-voice-op-codes) payload, we can proceed to the final step of voice connections, which entails establishing and handshaking a UDP connection for voice data. First, we open a UDP connection to the same endpoint we originally received in the [Voice Server Update](#DOCS_GATEWAY/voice-server-update) payload, combined with the port we received in the Voice Ready payload. If required, we can now perform an [IP Discovery](#DOCS_VOICE_CONNECTIONS/ip-discovery) using this connection. Once we've fully discovered our external IP and UDP port, we can then tell the voice websocket what it is, and start receiving/sending data. We do this using the Select Protocol Payload.
 
 >warn
-> The plain mode is deprecated and will be removed soon. All data should be sent using a supported encryption method (right now only `xsalsa20_poly1305`).
+> The plain mode is deprecated and will be removed soon. All data should be sent using a supported encryption method—right now only `xsalsa20_poly1305`.
```

Comment:
Lets change this to say no longer supported, we'll be removing it soon and this has been deprecated since I wrote these docs (basically).